data/constants/keycodes/keycodes_0.0.3_basic.hjson

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
needs a `keycodes_0.0.3.hjson` to properly generate new keycodes based on the basic hjson

Reply:
ok.  i see what it does.   also generates a new macro, so leveraging that for `IS_APPLE_FN_KEYCODE`.